wp-content/themes/core/templates/archive.html

```diff
@@ -13,11 +13,10 @@
 		<!-- wp:post-terms {"term":"category"} /-->
 		<!-- wp:post-excerpt /-->
 	<!-- /wp:post-template -->
-	<!-- wp:query-pagination -->
-	<div class="wp-block-query-pagination">
+	<!-- wp:query-pagination {"paginationArrow":"chevron","layout":{"type":"flex","justifyContent":"center"}} -->
 		<!-- wp:query-pagination-previous /-->
+		<!-- wp:query-pagination-numbers /-->
 		<!-- wp:query-pagination-next /-->
-	</div>
```

Comment:
Just a thought: Could we make the query-pagination a template part and include it in each of these templates? That would keep the config & styles consistent in each template, I think?

Reply:
Good call - I've moved the pagination blocks to a template part & updated the templates that use it. This will allow us to keep our pagination consistent across different templates.